Compilation error

Hi,

I am trying to compile grpc on Mac and I get the following error:

[LD]      Linking libs/opt/libgrpc++.dylib
Undefined symbols for architecture x86_64:
  "_grpc_call_set_credentials", referenced from:
      grpc::SecureCredentials::ApplyToCall(grpc_call*) in secure_credentials.o
  "_grpc_server_register_completion_queue", referenced from:
      grpc::Server::Server(grpc::ThreadPoolInterface*, bool, int) in server.o
      grpc::ServerBuilder::BuildAndStart() in server_builder.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [libs/opt/libgrpc++.dylib] Error 1

--- a/Makefile
+++ b/Makefile
@@ -262,6 +262,8 @@ LDFLAGS += -fPIC
 endif
 
 INCLUDES = . include $(GENDIR)
+LDFLAGS += -Llibs/$(CONFIG)
+
 ifeq ($(SYSTEM),Darwin)
 ifneq ($(wildcard /usr/local/ssl/include),)
 INCLUDES += /usr/local/ssl/include

--- a/templates/Makefile.template
+++ b/templates/Makefile.template
@@ -276,6 +276,8 @@ LDFLAGS += -fPIC
 endif
 
 INCLUDES = . include $(GENDIR)
+LDFLAGS += -Llibs/$(CONFIG)
+
 ifeq ($(SYSTEM),Darwin)
 ifneq ($(wildcard /usr/local/ssl/include),)
 INCLUDES += /usr/local/ssl/include

